multilabel_classification/pytorch_network_training.ipynb

```diff
@@ -0,0 +1,1064 @@
+    "        model.apply(reset_weights);\n",
+    "    dataset_train = DatasetMultiLabel(df = df.loc[train_index])\n",
+    "    dataset_val = DatasetMultiLabel(df = df.loc[test_index])\n",
+    "    loader_train = create_loader(\n",
+    "        dataset_train,\n",
+    "        input_size=data_config[\"input_size\"],\n",
+    "        batch_size=64,\n",
+    "        is_training=True,\n",
+    "        mean=data_config[\"mean\"],\n",
+    "        std=data_config[\"std\"],\n",
+    "       interpolation=data_config[\"interpolation\"],\n",
+    "    )\n",
+    "    loader_val = create_loader(\n",
+    "        dataset_val,\n",
+    "        input_size=data_config[\"input_size\"],\n",
+    "        batch_size=64,\n",
+    "        is_training=False,\n",
+    "        mean=data_config[\"mean\"],\n",
+    "        std=data_config[\"std\"],\n",
+    "        interpolation=data_config[\"interpolation\"],\n",
+    "\n",
+    "    )\n",
+    "    model.fit(loader_train,loader_val,num_epochs=40)\n",
+    "    checkpoint = torch.load(\"weights_model/model_best.pth.tar\")\n",
+    "    model.load_state_dict(checkpoint['state_dict'])\n",
+    "    pred_val = model.predict_proba(loader_val)\n",
+    "    df_pred = create_df(pred_val,dataset_val)\n",
+    "    df_pred.to_csv(str(ct)+\"_fold.csv\",index=False)\n",
+    "    ct+=1\n",
+    "    \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<module 'os' from '/usr/lib/python3.8/os.py'>"
+      ]
+     },
+     "execution_count": 20,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "os"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dfl = []\n",
+    "for i in range(1,num_splits+1):\n",
+    "    dfl.append(pd.read_csv(str(i)+\"_fold.csv\"))\n",
+    "    \n",
+    "\n",
+    "cols = dfl[0].columns[1:]\n",
+    "\n",
+    "df_pred = pd.concat(dfl,axis=0)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pred_probs = df_pred[cols].to_numpy()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "np.save(open('pred_probs.npy','wb'),pred_probs)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 36,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = list(df_pred['image_loc'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 38,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<module 'pickle' from '/usr/lib/python3.8/pickle.py'>"
+      ]
+     },
+     "execution_count": 38,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "pickle.dump(data,open(\"data.p\",'wb'))"
+   ]
```

Comment:
these can all be one cell that is used to save all the results.

Reply:
Done: https://github.com/cleanlab/examples/pull/27/commits/28f62b8528fdc7aec2fd87bca2d5da8e142b7b1e